src/client/pages/profile/profileview.jsx

```diff
@@ -0,0 +1,193 @@
+import React from "react";
```

Comment:
What about rename this file to `view.jsx` to reduce redundancy?

Reply:
Got it. Fixed!